Importer/CsvImporter.php

```diff
@@ -183,7 +170,7 @@ protected function importData(array $parsedData, $dataIsAssociativeArray, $flush
             return;
         }
 
-        if (!$flush || $this->log->containErrors()) {
+        if (!$flush || $this->log->getErrorCount() > 0) {
```

Comment:
The thing is, that if we have the function 

`getErrorCount`,

we can very easily check if it contains errors. 

**But** the other way around is not a thing, so If we decide to have the method `containErrors` and for whatever reason we need the count of the errors, how would we do this?

Long story short, `containErrors` is indeed more readable, **but** `getErrorCount` more useful. Especially since we know that we want to use this error count on the data interpreter. Otherwise we would to have to extend the `CsvLog` to also include a function called `getErrorCount`